src/main/java/CoreEntities/Player/Player.java

```diff
@@ -20,7 +20,7 @@ public Player(String name, LetterRack rack, int score) {
         this.rack = rack;
         this.score = score;
     }
-    abstract boolean makeMove(String word);
+    abstract boolean makeMove(String word, int start_x, int end_x, int start_y, int end_y);
```

Comment:
I'm wondering if you should abstract a method with this many parameters. If something has that many specific values. I think implementing an interface in a specific class that meets what the boolean needs to check?

Reply:
I do not think so. Since it is computer player, this method will only be called by generateMove